### ISSUE TITLE
Insufficient memory to run certain applications in Windows 3.11 with preferred DOSBox-X configuration

### ISSUE THREAD

**Issue by BridgeHeadland** [joncampbell123/dosbox-x#3942]
### Describe the bug

[dosbox-x.reference_USER.txt](https://github.com/joncampbell123/dosbox-x/files/10458347/dosbox-x.reference_USER.txt)
[dosbox-x_USER.txt](https://github.com/joncampbell123/dosbox-x/files/10458349/dosbox-x_USER.txt)
[dosbox-x.reference.full_USER.txt](https://github.com/joncampbell123/dosbox-x/files/10458350/dosbox-x.reference.full_USER.txt)
![image](https://user-images.githubusercontent.com/85258509/213491227-702e065b-689a-40f5-b0b7-031a340a9dec.png)
In any case, I have problems running Creative MIDI. I want to hear MIDI music, with presets/instruments from my custom soundfonts, and soundfonts that I have found around.
The main question is where should I reconfigure so that I can use software like Creative MIDI?
I'm wondering if adding the soundfont as a value under midiconfig and/or fluid.soundfont could be causing me to not get Creative MIDI to run and yet even if I reconfigure the MIDI section back to the original default I still get not running Creative Midi.

### Steps to reproduce the behaviour

Test out Creative MIDI or similar in Windows 3.11, if you have it. Reconfigure to the configurations shown in the TXT files and test Creative MIDI again. If that doesn't work, reconfigure the mididevice from default to fluidsynth, and add the soundfont, which is in the DOSBox-X directory, under midiconfig and fluid.soundfont. Test out Creative MIDI again.

### Expected behavior

I expect the specific Windows 3.11 software to work properly under the closest possible configuration to the preferred configuration.

### What operating system(s) this bug have occurred on?

Windows 10 Home, 21H2, 19044.2364

### What version(s) of DOSBox-X have this bug?

DOSBox-X, Vistual Studio Build, WIN64, 20230117083128

### Used configuration

```ini
I perceive CONF files as too large, so I paste them under Describe the bug, as TXT files.
```

### Have you checked that no similar bug report(s) exist?

- [X] I have searched and didn't find any similar bug report.

### Code of Conduct & Contributing Guidelines

- [X] I agree to follow the code of conduct and the contributing guidelines.

**Comment by rderooy** [comment]
Those are the default configs, which you can modify for specific applications. They are NOT the recommended configs for Windows 3.1x. Please check the DOSBox-X wiki for the recommended Win3.1x configs.

**Comment by BridgeHeadland** [comment]
I have solved the problem!
I shouldn't put ver 8.0 in the DOS section like I used to. From now on, I set it to auto. By the way, it solved the problem of launching software through Norton Commander.

**Comment by joncampbell123** [comment]
Ver 7 0 or higher enables LFN if wanted, and 7 10 or higher enables FAT32 emulation. Could that have caused this issue? What if you set ver 8 0 and then leave the LFN option off?

**Comment by NebularNerd** [comment]
> I have solved the problem! I shouldn't put ver 8.0 in the DOS section like I used to. From now on, I set it to auto. By the way, it solved the problem of launching software through Norton Commander.

Glad to hear you got NC working, could you close #3759 to help the mods with housekeeping please. :)
